src/get-started/huff-by-example/README.md

```diff
@@ -305,9 +305,9 @@ Functions can accept arguments to be "called" inside the macro or passed as a re
 
 Several builtin functions are provided by the Huff compiler:
 #### `__FUNC_SIG(<func_def|string>)`
-At compile time, the invocation of `__FUNC_SIG` is substituted with the function selector of the passed function definition or string. If a string is passed, it must represent a valid event i.e. `"test(address, uint256)"`
+At compile time, the invocation of `__FUNC_SIG` is substituted with the function selector of the passed function definition or string. If a string is passed, it must represent a valid function i.e. `"test(address, uint256)"`
 
-#### `__EVENT_HASH(<func_def|string>)`
+#### `__EVENT_HASH(<event_def|string>)`
```

Comment:
wrt the example below beginning on line 325, the first time I read through it I was a little thrown off because of the Jump Table when we hadnt talked about Jump tables yet. 

 I propose removing CONDITIONAL and CONDITIONAL_TEST from the first example below .  Let it just be a very concise example of EVENT_HASH, FN_SIG --- the following example will include tablestart/size after we explain tables.  Anyways, just my opinion.  Wyt @clabby @abigger87 ?
 
 Also what's PC?  I couldnt find it defined anywhere? 🤔

Reply:
#25